ipatests/pytest_ipa/integration/tasks.py

```diff
@@ -570,7 +570,7 @@ def install_client(master, client, extra_args=[], user=None,
 
     args.extend(extra_args)
 
-    if is_fips_enabled(client) and 'ad' in master:
+    if is_fips_enabled(client) and getattr(master, 'ad_domains', False):
```

Comment:
The ad_domains attribute is defined in master.config, not in master:
```
(Pdb) type(master)
<class 'ipatests.pytest_ipa.integration.host.Host'>
(Pdb) getattr(master, 'ad_domains', False)
False
(Pdb) getattr(master.config, 'ad_domains', False)
[<ipatests.pytest_ipa.integration.config.Domain object at 0x7f852f149a30>]
```

Reply:
fixed, thanks! I was confused as there is no explicit `config` being assigned to the host, only to domain. 
I suspect this comes from `ipa-test-task` binary.